src/tenants/planet/LeaderBoard/components/Stats.tsx

```diff
@@ -1,4 +1,4 @@
-import { Modal } from '@material-ui/core';
+import { Modal } from '@mui/material';
```

Comment:
As in other places, font size seen in the Modal is larger. Zoom level for both screens is the same.
![Screenshot 2022-05-09 at 10 42 25 AM](https://user-images.githubusercontent.com/44917347/167344763-6be6badc-09f6-4b20-a8a2-6892fa973867.png)

Reply:
Fixed Here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/899022f498edbef1e2c102fc2fae950fb381618a

Reply:
Resolved in #1467